next.config.js

```diff
@@ -1,4 +1,6 @@
 /** @type {import('next').NextConfig} */
-const nextConfig = {}
+const nextConfig = {
+  reactStrictMode: false,
```

Comment:
Just for curiosity, why do you disabled the React Strict mode?

Reply:
I disabled it because otherwise the components render twice (this is the default mode in development in order to find possible problems in the application) and some times it is not worth it.

Reply:
Good to know, thank you.